github/orgs_projects.go

```diff
@@ -12,7 +12,7 @@ import (
 
 // ListProjectsOptions specifies the optional parameters to the
 // OrganizationsService.ListProjects method.
-type ListProjectsOptions struct {
+type ProjectsListOptions struct {
```

Comment:
Ah, I see.

I think it should be in one of the two files, either `repos_projects.go` or `orgs_projects.go`. It doesn't really matter which of the two.

You could move it to `repos_projects.go` since that one feels a little more "primary", but either way is completely fine. Thanks. :)